prototype/core/api/src/main/java/org/eclipse/sensinact/core/model/ResourceBuilder.java

```diff
@@ -85,20 +85,21 @@
     ResourceBuilder<B, T> withAction(List<Entry<String, Class<?>>> namedParameterTypes);
 
     /**
-     * Set a getter function to be called
-     *
-     * @param getter
-     * @return
+     * This resource has a dynamic get behavior. Can't work on resource with an action.
      */
-    ResourceBuilder<B, T> withGetter(Supplier<T> getter);
+    ResourceBuilder<B, T> withGetter();
```

Comment:
Should we have a boolean version as well so that people can say `withGetter(false)` when copying another model?